cmd/cardinality.go

```diff
@@ -82,6 +86,7 @@ type seriesTable struct {
 	seriesScrapeText scrape.SeriesScrapeText
 	loading          bool
 	searchingMetrics bool
+	searchingLabels  bool
```

Comment:
nit: let me know if you want to tackle this in this PR. but we could have a Enum to tell which type of search we are using, since later we can add more search types, but not necessary.